src/main/java/mate/academy/Main.java

```diff
@@ -19,7 +20,6 @@ public static void main(String[] args) {
                     .thenAccept(userData -> System.out.println("Processed: " + userData));
         }
 
-        // Wait for all futures to complete
         CompletableFuture.allOf(futures).join();
         executor.shutdown();
```

Comment:
It's a good practice to shut down the ExecutorService within a try-finally block to ensure that it is shut down even if an exception occurs. This avoids potential resource leaks.